### PR TITLE
refactor!: start deprecation of `estimate_betweenness()`, `estimate_edge_betweenness()`, `estimate_closeness()`

### PR DESCRIPTION
**Pull request by maelle**
Part of #844 (the other part being making PRs to revdeps)

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -914,6 +914,7 @@ importFrom(graphics,symbols)
 importFrom(graphics,text)
 importFrom(graphics,xspline)
 importFrom(graphics,xyinch)
+importFrom(lifecycle,deprecated)
```
Comment (krlmlr) [inline NAMESPACE:917]: Are we using this?

Will this conflict with our own `deprecated()` function?

File: R/centrality.R [hunk 2]
```diff
@@ -19,20 +19,28 @@
...
+#' @inheritParams betweenness
 #' @export
-estimate_betweenness <- function(graph, vids = V(graph), directed = TRUE, cutoff, weights = NULL, nobigint = TRUE) {
-  if (!missing(nobigint)) {
-    warning("'nobigint' is deprecated since igraph 1.3 and will be removed in igraph 1.4")
-  }
+estimate_betweenness <- function(graph, vids = V(graph), directed = TRUE, cutoff, weights = NULL) {
+
+  lifecycle::deprecate_soft(
+    "1.5.0",
```
Comment (krlmlr) [inline R/centrality.R:33]:
```suggestion
    "1.6.0",
```

We may need to change to 2.0.0 later.

File: _pkgdown.yml [hunk 3]
```diff
@@ -237,6 +237,9 @@ reference:
   - layout.spring
   - layout.svd
   - "%>%"
+  - estimate_betweenness
```
Comment (krlmlr) [inline _pkgdown.yml:240]: Should we use `@keywords internal` in the documentation instead?

**Review by krlmlr — COMMENTED** [review]
Thanks, looks good!

File: NAMESPACE (hunk 1 above)
Comment (maelle) [reply NAMESPACE]: that came from `usethis::use_lifecycle()`, now removed

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**